index.html

```diff
@@ -555,30 +555,65 @@ <h2>
         mode</a>.
       </p>
       <p>
-        Each <a>display mode</a>, except <code>browser</code>, has a
-        <dfn>fallback display mode</dfn>, which is the <a>display mode</a> that
-        the user agent can try to use if it doesn't support a particular
-        <a>display mode</a>. If the user agent does support a <a>fallback
-        display mode</a>, then it checks to see if it can use that <a>display
-        mode</a>'s <a>fallback display mode</a>. This creates a fallback chain,
-        with the <a>default display mode</a> (<code>browser</code>) being the
-        last item in the chain.
+        The <dfn>fallback chain</dfn> for:
```

Comment:
Historical note: The old text here was wrong (or rather, nonsensical):

> If the user agent does support a fallback display mode, then it checks to see if it can use that display mode's fallback display mode.

This should have read "If the user agent does **not** support a **given** display mode, then it checks to see if it can use that display mode's fallback display mode."

This new text will help clarify it greatly. It might be worth adding a note to the commit message that this actually fixes broken text around how to deal with an unsupported display mode, even though we don't expect it to change implementations since we're preserving the intention behind the old broken text.